qa/standalone/scrub/osd-scrub-test.sh

```diff
@@ -503,10 +503,10 @@ function TEST_just_deep_scrubs() {
     # unset the 'no deep-scrub'. Deep scrubbing should start now.
     ceph osd unset nodeep-scrub || return 1
     sleep 5
-    declare -A expct_qry_duration=( ['query_last_duration']="0" ['query_last_duration_neg']="not0" )
+    declare -A expect_qry_duration=( ['query_last_duration']="0" ['query_last_duration_neg']="not0" )
     sc_data_2=()
     echo "test counter @ should be higher than before the unset: " ${sc_data_2['query_scrub_seq']}
-    wait_any_cond $pgid 10 $saved_last_stamp expct_qry_duration "WaitingAfterScrub " sc_data_2 || return 1
+    wait_any_cond $pgid 10 $saved_last_stamp expect_qry_duration "WaitingAfterScrub " sc_data_2 || return 1
```

Comment:
Leave this alone, shortening in code is not uncommon, we gain nothing by changing it.